src/discovery/local-discovery.js

```diff
@@ -25,10 +25,10 @@ export const ERR_DUPLICATE = 'Duplicate connection'
  */
 export class LocalDiscovery extends TypedEmitter {
   #identityKeypair
+  #name = randomBytes(8).toString('hex')
```

Comment:
nit: do you think it will it ever be useful to provide a custom name? can consider a constructor opt for this if so

Reply:
I think it might be useful in future but not right now, so I'm going to skip it for now.